howto/manage/setup-tls-for-appliance.md

```diff
@@ -0,0 +1,85 @@
+The Anbox Cloud Appliance uses the self-signed certificate to provide HTTPS services. If you want to serve the appliance over HTTPS using a valid SSL/TLS certificate, this document will walk you through the steps in generating and installing a valid SSL/TLS certificate on Anbox Cloud Appliance.
+
+[note type="information" status="Note"]This document assumes you have Anbox Cloud Appliance installed. If you haven't, please follow the [instruction](https://discourse.ubuntu.com/t/install-the-anbox-cloud-appliance/22681) to do so.[/note]
```

Comment:
```suggestion
[note type="information" status="Note"]This document assumes you have the Anbox Cloud Appliance installed. If you haven't, follow the [instructions](https://discourse.ubuntu.com/t/install-the-anbox-cloud-appliance/22681) to do so.[/note]
```

(We shouldn't use "please" in the documentation.)

Reply:
Okay, will fix.